docs/content/development/extensions-contrib/influx.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: doc_page
+---
+
+# InfluxDB Line Protocol Parser
+
+To use this extension, make sure to [include](../../operations/including-extensions.html) `druid-influx-extensions`.
+
+This extension enables Druid to parse [InfluxDB Line Protocol](https://docs.influxdata.com/influxdb/v1.5/write_protocols/line_protocol_tutorial/), a popular text-based timeseries metric serialization format. 
+
+## Line Protocol
+
+A typical line looks like this:
+
+```cpu,application=dbhost=prdb123,region=us-east-1 usage_idle=99.24,usage_user=0.55 1520722030000000000```
+
+which contains four parts:
+  - measurement: A string indicating what kind of measurement is represented (e.g. cpu, network, web_requests)
```

Comment:
I would stick with "name" over "kind" here. It's also the nomenclature in the Influx docs that you linked above.